concent_api/core/tests/utils.py

```diff
@@ -197,13 +202,57 @@ def _test_database_objects(
         task_id                     = None,
         receive_delivered_status    = None,
     ):
-        self.assertEqual(StoredMessage.objects.last().type,           last_object_type.TYPE)
-        self.assertEqual(StoredMessage.objects.last().task_id,        task_id)
+        self.assertEqual(StoredMessage.objects.order_by('timestamp').last().type,      last_object_type.TYPE)
+        self.assertEqual(StoredMessage.objects.order_by('timestamp').last().task_id,   task_id)
 
         if receive_delivered_status is not None:
             self.assertEqual(ReceiveStatus.objects.last().delivered,        receive_delivered_status)
             self.assertEqual(ReceiveStatus.objects.last().message.task_id,  task_id)
 
+    def _test_subtask_state(
+        self,
+        task_id,
+        subtask_id,
+        subtask_state,
+        provider_key,
+        requestor_key,
+        next_deadline                   = None,
+        has_task_to_compute             = None,
+        has_report_computed_task        = None,
+        has_ack_report_computed_task    = None,
+        has_reject_report_computed_task = None,
```

Comment:
Might be better to have a parameter that is a list (or a set) of field names that are supposed not to be `None`. This would shorten the parameter list and we could process them here in a loop.